collections/_article/release-candidate-godot-4-3-rc-1.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Release candidate: Godot 4.3 RC 1"
+excerpt: ""
+categories: ["pre-release"]
+author: "Rémi Verschelde"
+image: /storage/blog/covers/release-candidate-godot-4-3-rc-1.webp
+image_caption_title: "ColdRidge"
+image_caption_description: "A game by Frog Collective"
+date: 2024-07-25 15:00:00
```

Comment:
We need to start adding the proper time here, but it can be edited after the review. @winston-yallow do you know the local time of the server?

Reply:
Is it important for the date to be accurate? We've always picked roughly a time that's before publishing date on the same day, which seems good enough. I guess mostly RSS feeds might see it as "published 3 hours ago" even though it was published now.

As for local time, I'm assuming UTC here.

Still, if we can infer it at build time, that could work. But since we rebuild everything on each commit, we'd need that information to be committed somehow I guess?